packages/auth/package.json

```diff
@@ -10,18 +10,22 @@
     "lint": "eslint ."
   },
   "peerDependencies": {
-    "better-auth": "^1.2.8",
+    "better-auth": "1.2.9-beta.8",
     "server-only": "^0.0.1"
   },
   "dependencies": {
+    "@googleapis/people": "^3.0.9",
+    "@microsoft/microsoft-graph-client": "^3.0.7",
     "@repo/db": "workspace:*",
-    "@repo/env": "workspace:*"
+    "@repo/env": "workspace:*",
+    "google-auth-library": "^9.15.1"
```

Comment:
Don't think we need `@googleapis/people`, or `google-auth-library` anymore, right?